README.md

```diff
@@ -990,6 +990,12 @@ IEEE Trans. Image Process. (TIP), 33:1826-1837, 2024. [PDF](https://arxiv.org/pd
 - <a name="Zhang24tip"></a>Zhang, P., Liu, H., Ge, Z., Wang, C., Lam, E. Y.,   
 *[Neuromorphic Imaging with Joint Image Deblurring and Event Denoising](https://doi.org/10.1109/TIP.2024.3374074)*,  
 IEEE Trans. Image Process. (TIP), 2024. [PDF](https://arxiv.org/pdf/2309.16106.pdf), [Code](https://github.com/pz-even/ni-joint)
+- <a name="Muglikar23arxiv"></a>Zou.R, Muglikar.M, Messikommer.N, Scaramuzza. D,   
```

Comment:
Spaces and commas between authors and their initials, please

Reply:
fixed